src/app/components/members/Member.js

```diff
@@ -4,17 +4,25 @@ import { Col } from 'react-bootstrap'
 export default class Member extends Component {
   render () {
     const style = { backgroundImage: 'url(' + this.props.img + ')' }
-    const email = 'mailto:' + this.props.mail
-    const twitterLink = 'https://twitter.com/' + this.props.twitter
-    const gitHubLink = 'https://github.com/' + this.props.github
-
+    let mailLink
+    if (typeof this.props.mail !== 'undefined') {
+      mailLink = <a target='_blank' href={'mailto:' + this.props.mail} className='social-icon fa fa-envelope-o' />
+    }
+    let twitterLink
+    if (typeof this.props.twitter !== 'undefined') {
+      twitterLink = <a target='_blank' href={'https://twitter.com/' + this.props.twitter} className='social-icon fa fa-twitter' />
+    }
+    let gitHubLink
+    if (typeof this.props.github !== 'undefined') {
```

Comment:
In my opinion, checking if it is undefined is more reliable, and more accurate for this case.
But, of course, the two ways work